nncf/torch/quantization/layers.py

```diff
@@ -428,14 +429,31 @@ def _prepare_fq_export_quantization(self, x: torch.Tensor):
             levels = level_high - level_low + 1
         return x, levels, input_low, input_high
 
-    def _prepare_qdq_export_quantization(self, x: torch.Tensor):
+    def _prepare_qdq_export_quantization(self, x: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor, int]:
         x, level_high, level_low, input_low, input_high = self._prepare_export_quantization(x)
         with no_jit_trace():
             y_scale, y_zero_point = get_scale_zp_from_input_low_input_high(level_low,
                                                                            level_high,
                                                                            input_low,
                                                                            input_high)
-        return x, y_scale, y_zero_point
+            possible_axes = self._possible_per_channel_dimensions()
+            if len(possible_axes) > 1:
+                raise RuntimeError(
+                    f"Impossible to determine the per-channel axis for a scale shape {self.scale_shape} - "
+                    f"more than one dimension is >1")
+            if not possible_axes:
+                # Impossible to determine proper axis for per-channel quantization because we have
```

Comment:
The exception should be raised in this case as well because it should indicate that there is a logic in FQ insertion logic.